test/unit.js

```diff
@@ -66,4 +72,27 @@ describe('Get documents', function() {
       done();
     });
   });
-});
\ No newline at end of file
+});
+
+describe('Register Channel: ', function(){
```

Comment:
I just realized that these tests depend on you running an active instance of OpenHim - that kind of makes them integration tests. Should I leave them, or try and mock them using nock?